From e317d5277e34948ae7048cb5e48309e0288e8df3 Mon Sep 17 00:00:00 2001
From: Nevena Kotlaja <kotlaja@google.com>
Date: Thu, 29 Feb 2024 12:07:18 -0800
Subject: [PATCH] Remove implicit dependencies of
 `_allowlist_function_transition`.

Rules are not required to have an implicit dependencies on the transition allowlist since Bazel knows where the file is.

PiperOrigin-RevId: 611552493
---
 centipede/instrument.bzl         | 3 ---
 centipede/testing/build_defs.bzl | 3 ---
 2 files changed, 6 deletions(-)

diff --git a/centipede/instrument.bzl b/centipede/instrument.bzl
index 605ce905..8d223881 100644
--- a/centipede/instrument.bzl
+++ b/centipede/instrument.bzl
@@ -78,9 +78,6 @@ target cc_binary.
             cfg = strip_instrumentation_transition,
             mandatory = True,
         ),
-        "_allowlist_function_transition": attr.label(
-            default = "@bazel_tools//tools/allowlists/function_transition_allowlist",
-        ),
     },
     executable = True,
 )
diff --git a/centipede/testing/build_defs.bzl b/centipede/testing/build_defs.bzl
index ec3878f0..bb2bae05 100644
--- a/centipede/testing/build_defs.bzl
+++ b/centipede/testing/build_defs.bzl
@@ -104,9 +104,6 @@ __sancov_fuzz_target = rule(
             executable = True,
             mandatory = True,
         ),
-        "_allowlist_function_transition": attr.label(
-            default = "@bazel_tools//tools/allowlists/function_transition_allowlist",
-        ),
         "sancov": attr.string(),
     },
     executable = True,
